tests/src/Kernel/Access/WikiTest.php

```diff
@@ -60,6 +61,7 @@ class WikiTest extends KernelTestBase {
 
   /**
    * {@inheritdoc}
+   * @expectedException \LogicException
```

Comment:
lets dedicate own method (e.g. `testDefaultRolesSave`) to assert the exception, and keep the setup() error free